P5/Source/Specs/att.datcat.xml

```diff
@@ -7,51 +7,221 @@ $Date$
+      were not anchored in a single point of an established reference taxonomy of morphosyntactic
+      data categories. In the case at hand, the string <q><ref
+        target="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545"
+        >http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545</ref></q> is both
+      a persistent identifier of the data category in question, as well as the address of the place where a shared definition
+      of <q>common noun</q> is provided.</p>
+    <p>While the symbols <q>NN</q>, <q>NN0</q>, and many others (often coming from languages other than
+      English) are implicitly members of the container category <q>part of speech</q>, it is sometimes
+      useful not to rely on such an implicit relationship but rather use an explicit identifier
+      for that data category, to distinguish it from other morphosyntactic data categories, such
+      as <soCalled>gender</soCalled>, <soCalled>tense</soCalled>, etc. For that purpose, the above example uses the <att>datcat</att>
+      attribute to reference a definition of <soCalled>part of speech</soCalled>. The reference taxonomy in this example is the 
+      <ref target="https://www.clarin.eu/content/clarin-concept-registry">CLARIN Concept
+        Registry</ref>.</p>
+    
+  </exemplum>
+  <exemplum xml:lang="en">
+    <p>If the feature structure markup exemplified above is to be repeated many times in a single
+      document, it is much more efficient to gather the persistent identifiers in a single place and to
+      only reference them, implicitly or directly, from feature structure markup. The following
+      example is much more concise than the one above and relies on the concepts of feature structure declaration and
+      feature value library, discussed in chapter <ptr target="#FS"/>.</p>
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
       <fs>
-        <f name="POS" dcr:datcat="http://www.isocat.org/datcat/DC-1345" fVal="#commonNoun" dcr:valueDatcat="http://www.isocat.org/datcat/DC-1256"/>
+        <f name="POS" fVal="#commonNoun"/>
+        <!-- ... -->
       </fs>
     </egXML>
+    <p>The assumption here is that the relevant feature values are collected in a place that the
+      annotation document in question has access to — preferably, a single document per linguistic
+      resource, for example a <gi>fsdDecl</gi> that is XIncluded as a sibling of <gi>text</gi> or a
+      child of <gi>encodingDesc</gi>; a <gi>taxonomy</gi> available resource-wide (e.g., in a shared
+      header) is also an option. The example below presents a <gi>fvLib</gi> element that collects the
+      relevant feature values (most of them omitted).</p>
+  </exemplum>
+  <exemplum xml:lang="en">
+    <p>This example shows one way of encoding a <hi rend="bold">tagset</hi>, i.e., an established inventory of values of
```

Comment:
`<hi>`? Should be `<term>`, no?

Reply:
yes, I don't know what I was thinking when encoding that.